### ISSUE TITLE
one trial took 7 hours and block in the 32th epoch

### ISSUE THREAD

**Issue by AsmaBenBrahem** [keras-team/keras-tuner#527]
Hello everybody, I am using keras tuner version 1.0.1 with tensorflow 2.0.0 and keras 2.3.1 to fine tune the learning rate, batch-size and the number of epoch.
![image](https://user-images.githubusercontent.com/48836506/116896193-e32ed300-ac2b-11eb-88d0-e7077082f0c7.png)
![image](https://user-images.githubusercontent.com/48836506/116896296-03f72880-ac2c-11eb-83a9-7d4862d18208.png)
![image](https://user-images.githubusercontent.com/48836506/116902080-a0bcc480-ac32-11eb-80da-1c4b1db8b932.png)

![image](https://user-images.githubusercontent.com/48836506/116896763-77993580-ac2c-11eb-9ee3-0a827ebaf569.png)
as you can see only one trial with learning rate=0.01, batch size=8 and number of epochs=80 took 7 hours until only the 32th and the colab bans me from using the gpu ( the google account is brand new, only used it for this notebook) and the results are very poor.
while without keras tuner 100 epochs doesn't even take 2 hours for this batch size and this learning rate and the training iou already get the value of 0.78 and val_iou=0.3 from the first epoch while for the first epoch with keras tuner :
![image](https://user-images.githubusercontent.com/48836506/116898387-39047a80-ac2e-11eb-8b3f-915cde3e6e69.png)

 Can someone help me with this problem?
N.B I used the subclass provided here in the documentation https://github.com/keras-team/keras-tuner/issues/122

**Comment by AsmaBenBrahem** [comment]
solved by setting tensorflow version to 2.1
